pkg/controller/s3/bucketpolicy/bucketpolicy.go

```diff
@@ -44,6 +44,7 @@ const (
 	errDelete           = "failed to delete the policy for bucket"
 	errGet              = "failed to get BucketPolicy for bucket with name"
 	errUpdate           = "failed to update the policy for bucket"
+	errNotSpecified     = "failed to format bucketPolicy, no policyBody or jsonBody specified"
```

Comment:
Error string should be updated to reflect updated fields `rawPolicy` and `policy`